--- mcrypt-2.5.7/configure.in~	Mon Jun  4 16:35:31 2001
+++ mcrypt-2.5.7/configure.in	Wed Jun 13 23:22:43 2001
@@ -161,6 +161,4 @@
 AC_CHECK_FUNCS(lstat stat umask utime memmove bcopy getpwuid waitpid,,)
 
 
-AC_CONFIG_FILES([Makefile doc/Makefile src/Makefile intl/Makefile po/Makefile.in])
-AC_CONFIG_COMMANDS([default],[[sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile]],[[]])
-AC_OUTPUT
+AC_OUTPUT([Makefile doc/Makefile src/Makefile intl/Makefile po/Makefile.in])
